Package not registering its dependencies on test

This leads to errors like this:

``` python
Failure in test test_ievent (brasil.gov.agenda.tests.test_compromisso.ContentTypeTestCase)
Traceback (most recent call last):
  File "/home/hvelarde/collective/buildout.python/parts/opt/lib/python2.7/unittest/case.py", line 331, in run
    testMethod()
  File "/home/hvelarde/simples/idgb/src/brasil.gov.agenda/src/brasil/gov/agenda/tests/test_compromisso.py", line 123, in test_ievent
    self.assertTrue(IEvent.providedBy(self.compromisso))
  File "/home/hvelarde/collective/buildout.python/parts/opt/lib/python2.7/unittest/case.py", line 424, in assertTrue
    raise self.failureException(msg)
AssertionError: False is not true
```

--- a/docs/CHANGES.rst
+++ b/docs/CHANGES.rst
@@ -4,6 +4,9 @@ Changelog
 1.1.1 (unreleased)
 ------------------
 
+- Register explicitly plone.app.event dependency on configure.zcml.
+  [hvelarde]
+
 - Check if there is an image into leadimage field for migration.
   [bsuttor]
 

--- a/plone/app/contenttypes/configure.zcml
+++ b/plone/app/contenttypes/configure.zcml
@@ -8,6 +8,7 @@
            i18n_domain="plone.app.contenttypes">
 
   <includeDependencies package="." />
+  <include package="plone.app.event" />
   <include file="profiles.zcml" />
 
   <include package=".behaviors" />

